tern/analyze/common.py

```diff
@@ -555,9 +556,42 @@ def get_git_sha(path_to_toplevel):
     return sha_info
 
 
+def get_git_url(path_to_toplevel):
+    '''Given a absolute path to a git repository, return the project url.'''
+    command = ['git', 'remote', '-v']
+    url_info = '(not found)'
+    try:
+        output = subprocess.check_output(  # nosec
+            command, stderr=subprocess.DEVNULL, cwd=path_to_toplevel)
+        if isinstance(output, bytes):
+            lines = output.decode('utf-8').split('\n')
+            url_dict = {}
+            for line in lines:
+                # line structure is '<label>/t<url> (fetch)'
```

Comment:
Did you mean `<label>\t<url>`?

Reply:
Yes! The form of one line in the output is like `<label>\t<url> (fetch/push)`